src/coreclr/jit/llvm.cpp

```diff
@@ -166,11 +202,32 @@ llvm::Type* getLlvmTypeForCorInfoType(CorInfoType corInfoType) {
     }
 }
 
+// TODO: Seems like this is not a great idea, but when looking at a sigInfo from eeGetMethodSig we have CorInfoType(s) but when looking at lclVars we have LclVarDsc or var_type(s), it would be better if a common
+// denomiator was used, but I couldn't find one so introduced this method to workaround.
+CorInfoType toCorInfoType(var_types varType)
+{
+    switch (varType)
+    {
+        case var_types::TYP_BYREF:
+            return CorInfoType::CORINFO_TYPE_BYREF;
+        case var_types::TYP_LCLBLK: // TODO: what is this in the lvaTable?   Dont think it needs to be on the shadow stack, so map to int.
```

Comment:
`LCLBLK` is the outgoing args area, aka the bytes in the function's frame that the callee can use as their own. It is only used for that one local and for nothing else (honestly, we should probably just delete it to save on memory in VN's static arrays).

It is a pretty low-level detail of the calling convention whether you'll want to have outgoing args implemented explicitly or not, but if you do, the best match for it would be `CorInfoType::CORINFO_TYPE_VALUECLASS` (aka opague block of bytes).